samcli/lib/providers/sam_function_provider.py

```diff
@@ -445,6 +446,9 @@ def _build_function_configuration(
             codeuri = SamLocalStackProvider.normalize_resource_path(stack.location, codeuri)
 
         package_type = resource_properties.get("PackageType", ZIP)
+        if package_type == ZIP and resource_properties.get("Handler") is None:
```

Comment:
Should we also validate for an empty handler (ie. `Handler: ""`)? If that's also an invalid case then we can include that in this check.

Reply:
Yea thats a good call, ill add that in there as well